ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/documentation/devices/snmp.md

```diff
@@ -57,6 +57,7 @@ interface Management1
 | ------- | -------- | ---------- | ----- |
 | DC1_OPS | DC1 | All | Enabled |
 
+
```

Comment:
this extra space is caused by some template change. I will try to identify it.

Reply:
the extra spaces in the j2 of the doc, let me comment there

Reply:
Thanks for catching this! addressed in 81d0d63